resolve_march_native/engine.py

```diff
@@ -28,7 +28,7 @@ def _extract_tune_from_flags(flags):
 		for flag in flags:
 			if flag.startswith(prefix):
 				return (flag, flag[len(prefix):])
-		raise ValueError('No entry -tune=.. found in: %s' % ' '.join(sorted(flags)))
+		return (None, None)
```

Comment:
I support the general idea but I would rather make `_resolve_mtune` robust to that exception than returning `None`. Can you teach `_resolve_mtune` catching `ValueError` (or a new dedicated exception), instead? Should I take over?

Reply:
Yes, you are right, this might be the cleaner way. Give me a minute, I'll update my commit.

Reply:
Cool!